packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,223 @@
+- v0: Does not have equivalent component.
+
+## Anatomy
+
+![Anatomy](./etc/images/anatomy.png)
+
+### Avatar, PresenceBadge, and Avatar + PresenceBadge
+
+v8's Persona has a sub-component `PersonaCoin`, the equivalent in v9 is an Avatar. Currently Avatar is able to display a PresenceBadge, but this behavior will be deprecated in favor of Persona having this behavior.
+
+Persona has a slot for an `Avatar` and `PresenceBadge`, giving the user the ability to display an Avatar, a PresenceBadge, or Combine them into an Avatar with presence.
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+![Avatar](./etc/images/avatar.png)
+
+Persona with Avatar:
+
+```jsx
+<Persona name="Kevin Sturgis" primaryText="Kevin Sturgis" secondaryText="Software Engineer" />
+```
+
+![PresenceBadge](./etc/images/badge.png)
+
+Persona with PresenceBadge:
+
+```jsx
+<Persona presenceOnly presence={{ status: 'offline', outOfOffice: true }} primaryText="Kevin Sturgis" />
+```
+
+![Avatar and PresenceBadge](./etc/images/avatar_badge.png)
+
+Persona with Avatar + PresenceBadge:
+
+```jsx
+<Persona
+  name="Kevin Sturgis"
+  presence={{ status: 'offline', outOfOffice: true }}
+  primaryText="Kevin Sturgis"
+  secondaryText="Software Engineer"
+  tertiaryText="Offline"
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+> I will refer to the avatar and presence as content.
+
+There are three alignment variants:
+
+- after: Content on the left and text on the right.
+- before: Content on top and text on the bottom.
+- below: Content on the right and text on the left.
+
+There are 3 content variants:
+
+- Avatar
+- PresenceBadge
+- Avatar + PresenceBadge
+
+There are 2 sizing variants:
+
+- scaled: The content will be resized based on the number of text lines.
+- fixed: The content will remain the same size and the text lines will be styled based on the content's size.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+**Slots**
+
+- `root`: The root slot for Persona.
+- `avatar`: The Avatar to display and Persona's primary slot.
+- `presence`: The PresenceBadge to display.
+- `primaryText`: Primary text.
+- `secondaryText`: Secondary text.
+- `tertiaryText`: Tertiary text.
+- `quaternaryText`: Quaternary text.
+
+**Types**
+
+```ts
+export type PersonaSlots = {
+  root: NonNullable<Slot<'div'>>;
+
+  /**
+   * Avatar to display.
+   */
+  avatar: NonNullable<Slot<typeof Avatar>>;
+
+  /**
+   * PresenceBadge to display.
+   */
+  presence: Slot<typeof PresenceBadge>;
+
+  /**
+   * Primary text to display.
```

Comment:
It might be nice to have these doc comments be a little more descriptive. E.g.:
```suggestion
   * The first line of text, larger than other lines of text.
```